lib/src/repository/user_repository.dart

```diff
@@ -9,7 +9,7 @@ import 'gql_client.dart';
 
 class UserRepository {
   static Future<User> getUserDetails(Request request, {Link? graphLink}) async {
-    if (graphLink == null) graphLink = GraphLinkCreator.create(request);
+    graphLink = graphLink ?? GraphLinkCreator.create(request);
```

Comment:
not needed but I found out Dart can do null-aware assignment and can shorten lines like this to:
`graphLink ??= GraphLinkCreator.create(request);`